tools/cldr-code/src/main/java/org/unicode/cldr/util/VettingViewer.java

```diff
@@ -263,6 +263,34 @@ public enum VoteStatus {
         VoteResolver<String> getVoteResolver(CLDRLocale loc, String path);
     }
 
+    public static class EmptyUsersChoice implements UsersChoice<Organization> {
+        final PathHeader.Factory phf = PathHeader.getFactory();
+
+        @Override
+        public String getWinningValueForUsersOrganization(CLDRFile cldrFile, String path, Organization org) {
+            CLDRLocale loc = CLDRLocale.getInstance(cldrFile.getLocaleID());
+            return getVoteResolver(loc, path).getOrgVote(org);
+        }
+
+        @Override
+        public VoteStatus getStatusForUsersOrganization(CLDRFile cldrFile, String path, Organization org) {
+            CLDRLocale loc = CLDRLocale.getInstance(cldrFile.getLocaleID());
+            return getVoteResolver(loc, path).getStatusForOrganization(org);
+        }
+
+        @Override
+        public VoteResolver<String> getVoteResolver(final CLDRLocale loc, final String path) {
+            VoteResolver<String> r = new VoteResolver<>();
+            r.setLocale(loc, phf.fromPath(path));
```

Comment:
probably the VoteResolver needs additional initialization

Reply:
compare: STFactory.PerLocaleData#getResolverInternal, which calls setUsingKeywordAnnotationVoting, setBaseline, setBaileyValue

Reply:
setUsingKeywordAnnotationVoting can be simplified to be entirely private within VoteResolver, so it won't be necessary to duplicate that code in EmptyUsersChoice

setBaileyValue might be problematic since EmptyUsersChoice#getVoteResolver doesn't have the CLDRFile

unfortunately, setBaseline is very problematic since the corresponding code in STFactory has complex private dependencies:

```
            final ValueChecker vc = ERRORS_ALLOWED_IN_VETTING ? null : new ValueChecker(path);
            ...
            final String currentValue = diskData.getValueAtDPath(path);
            final Status currentStatus = calculateStatus(diskFile, diskFile, path);
            if (ERRORS_ALLOWED_IN_VETTING || vc.canUseValue(currentValue)) {
                r.setBaseline(currentValue, currentStatus);
                r.add(currentValue);
            }
```

Reply:
aside from having zero votes, we need EmptyUsersChoice to act the same as STUsersChoice -- otherwise it's meaningless to compare current stats with baseline